spec/factories/host.rb

```diff
@@ -2,7 +2,7 @@
   factory :host do
     sequence(:name)     { |n| "host_#{seq_padded_for_sorting(n)}" }
     sequence(:hostname) { |n| "host_#{seq_padded_for_sorting(n)}" }
-    vmm_vendor          "vmware"
+    vmm_vendor_display  "VMware"
```

Comment:
Yeah, remove the setter...I think it makes it a lot clearer which code needs which responsibility, and which code is doing the Wrong Thing :tm: